gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreEventHelper.java

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.hive.metastore;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.base.Optional;
+import com.google.common.collect.ImmutableMap;
+
+import gobblin.hive.HivePartition;
+import gobblin.hive.HiveTable;
+import gobblin.hive.spec.HiveSpec;
+import gobblin.metrics.event.EventSubmitter;
+
+
+/**
+ * Helper class to populate hive registration event in state.
+ */
+public class HiveMetaStoreEventHelper {
+  
+  public static final String SUCCESS_POSTFIX = "Succeed";
+  public static final String FAILED_POSTFIX = "Failed";
+  
+  public static final String DB_NAME = "DBName";
+  public static final String TABLE_NAME = "TableName";
+  public static final String ERROR_MESSAGE = "ErrorMessage";
+
+  // Path Registration
+  protected static void submitSuccessfulPathRegistration(EventSubmitter eventSubmitter, HiveSpec spec) {
+    eventSubmitter.submit("PathRegistration" + SUCCESS_POSTFIX,
```

Comment:
Please make `"PathRegistration"` a constant (same for all other strings in this class)